runtime/crab/src/migrations.rs

```diff
@@ -36,24 +23,56 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> Weight {
-	darwinia_balances::migration::migrate::<Runtime, RingInstance>();
-	darwinia_balances::migration::migrate::<Runtime, KtonInstance>();
+	// --- darwinia-network ---
+	use darwinia_staking::{StakingLedger, STAKING_ID};
+
+	let now = System::block_number();
```

Comment:
Use a hardcode number to test with fork-off-substrate.
Otherwise `now` is 1 in the new chain.